Apps/Admin/Client/Api/IAdminReportApi.cs

```diff
@@ -31,7 +31,7 @@ public interface IAdminReportApi
         /// </summary>
         /// <returns>A collection of <see cref="BlockedAccessRecord"/> records.</returns>
         [Get("/BlockedAccess")]
-        Task<IEnumerable<BlockedAccessRecord>> GetBlockedAccessReport();
+        Task<IEnumerable<BlockedAccessRecord>> GetBlockedAccessReportAsync();
```

Comment:
It wouldn't hurt to have it, but it was intentionally left out because we don't have anything to pass in, since these are in the Blazor front-end.

Reply:
https://www.youtube.com/watch?v=97gGYgZ8xU0&ab_channel=Codewrinkles

This may not be the only pattern of implementing cancellation tokens but it would appear at first glance that it requires some work.